.azure-pipelines.yml

```diff
@@ -448,7 +448,7 @@ jobs:
 
 - job: 's2_linux_snap_build'
   displayName: 'Linux snap build'
-  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  # condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
```

Comment:
Why we remove the condition ? we want to build a snap image for every branch ?

Reply:
That was for testing